lib/Dialect/Torch/Transforms/RefineTypes.cpp

```diff
@@ -1329,6 +1335,21 @@ ChangeResult TypeAnalyzer::visitAtenSoftmaxLikeOp(
   return getLatticeElement(op.getResult()).join(knowledge);
 }
 
+ChangeResult TypeAnalyzer::visitAten_SoftmaxOp(
+    Aten_SoftmaxOp op, ArrayRef<LatticeElement<ValueKnowledge> *> operands) {
+  auto input = operands[0]->getValue();
```

Comment:
This part of the code could be shared by a lot of operations. Can you add a helper like `ValueKnowledge getSameSizeAsInput(Operation *op, ArrayRef<LatticeElement<ValueKnowledge> *> operands)` and share the helper with other softmax like ops?

Reply:
Added a helper function.